src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -82,6 +83,9 @@ const RegistrationForm = () => {
         </div>
         <Button type="submit">Register</Button>
       </form>
+      <Link className="link text-center mt-3" to="/login">
```

Comment:
nit: new line above `Link` to match other component or remove new line in the other?